config/crd/bases/numaplane.numaproj.io_isbservicerollouts.yaml

```diff
@@ -238,17 +238,17 @@ spec:
                     description: UpgradingISBServiceStatus represents either the current
                       or otherwise the most recent "upgrading" isbservice
                     properties:
-                      assessmentEndTime:
-                        description: AssessmentEndTime indicates the time after which
-                          no more assessments will be performed
-                        format: date-time
-                        type: string
                       assessmentResult:
                         description: AssessmentResult described whether it's failed
                           or succeeded, or to be determined
                         type: string
-                      assessmentStartTime:
-                        description: AssessmentStartTime indicates the time at/after
+                      basicAssessmentEndTime:
+                        description: BasicAssessmentEndTime indicates the time after
+                          which no more assessments will be performed
+                        format: date-time
+                        type: string
+                      basicAssessmentStartTime:
```

Comment:
I renamed these to clarify it's just the "basic resource health check" we're assessing for